.github/workflows/cypress.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   cypress-run:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Should we trust on the latest like that? Or should we stick with some LTS release?

Reply:
I've checked this workflow git history and previously we used `latest`, hence why I used `ubuntu-latest` since this is just to execute some check scripts.

According to the deprecation notice, Ubuntu 24.04 moved to General Avaliability on February. We have these labels available: `ubuntu-latest`, `ubuntu-22.04`, `ubuntu-24.04`.

But to be on the safer side, I can use `ubuntu-22.04` instead.

Reply:
If we are able to make it work with 24.04, it should be a long period before we have to worry about it, instead of having to deal with failed builds because something changed on an STS version.

Reply:
The latest tag is pulling 24.04, so I believe that is safe. I'll replace the `ubuntu-latest` with `ubuntu-24.04`

<img width="1350" alt="Screenshot 2025-05-05 at 18 03 48" src="https://github.com/user-attachments/assets/43f71050-8f70-4f00-96dd-eab8ffe5b784" />